01.fizzbuzz/fizzbuzz.rb

```diff
@@ -1,16 +1,12 @@
-a=1
-
-20.times{
-
-if a%15 == 0 
-  puts "fizzbuzz"
-elsif a%5 ==0
-  puts "buzz"
-elsif a%3 ==0
-  puts "fizz"
-else 
-  puts a
+20.times do |n|
```

Comment:
```suggestion
1.upto(20) do |n|
```

のようにするとnが1から始まるようになります。